<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_ALEDataEdit</class>
 <widget class="QDialog" name="W_ALEDataEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑帧</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>当前帧</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="1,4,1">
      <item row="0" column="0">
       <widget class="QLabel" name="label_zeroFillChar">
        <property name="text">
         <string>当前资源</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>12</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>当前帧时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>0.01秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.02秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.03秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.04秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.10秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>0.50秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1.00秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.00秒</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>自定义</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox">
        <property name="suffix">
         <string>秒</string>
        </property>
        <property name="minimum">
         <double>0.010000000000000</double>
        </property>
        <property name="maximum">
         <double>100.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>W_ALEDataEdit</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
